modules/pages/client/components/Home.component.js

```diff
@@ -294,7 +294,11 @@ export default function Home({ user, isNativeMobileApp, photoCredits }) {
       </Board>
 
       {/* Footer */}
-      <Board className="board-primary board-inset home-footer" names="bokeh">
+      <Board
+        className="board-primary board-inset home-footer"
+        names="bokeh"
+        ignoreSmall={true}
```

Comment:
Nit; can be shortened to `ignoreSmall` without `={true}`.

Would something like `ignoreBackgroundOnSmallScreen` be clearer? It's mouthful tho... Up to you!

Reply:
Absolutely!